tests/integration/dogshell/test_dogshell.py

```diff
@@ -109,20 +99,13 @@ def match_permalink(out):
         out, err, return_code = self.dogshell(cmd, stdin=body)
 
         event_id = match_permalink(out)
-        assert event_id, out
-
-        # Add a bit of latency for the event to appear
-        time.sleep(self.wait_time)
```

Comment:
no need for the latency anymore?

Reply:
I use the new `dogshell_with_retry`, so yeah it's not needed here anymore